anvil_consortium_manager/tests/test_views.py

```diff
@@ -2858,10 +2858,10 @@ def test_notification_email(self):
         self.assertEqual(len(mail.outbox[0].to), 1)
         self.assertIn("test@example.com", mail.outbox[0].to)
 
-    @override_settings(ANVIL_ACCOUNT_VERIFY_NOTIFICATION_EMAIL=None)
+    @override_settings(ANVIL_ACCOUNT_ADAPTER="anvil_consortium_manager.tests.test_app.adapters.TestAccountAdapter")
```

Comment:
I think this is essentially duplicating the `def test_no_notification_email` test now, because the default account adapter should have account_verify_notification_email=None.